weeb/module.py

```diff
@@ -0,0 +1,145 @@
+            [],
+            [],)
+        tags = []
+        for tag in dic["tags"]:
+            if tag["type"] == "character":
+                characters.append(tag["name"])
+            elif tag["type"] == "parody":
+                parodies.append(tag["name"])
+            elif tag["type"] == "tag":
+                tags.append(tag["name"])
+            elif tag["type"] == "artist":
+                artists.append(tag["name"])
+            elif tag["type"] == "group":
+                groups.append(tag["name"])
+            elif tag["type"] == "language":
+                languages.append(tag["name"])
+            elif tag["type"] == "category":
+                categories.append(tag["name"])
+
+        characters = ["Characters", ",".join(str(e) for e in characters)]
+        parodies = ["Parodies", ",".join(str(e) for e in parodies)]
+        tags = ["Tags", ",".join(str(e) for e in tags)]
+        artists = ["Artists", ",".join(str(e) for e in artists)]
+        groups = ["Groups", ",".join(str(e) for e in groups)]
+        languages = ["Languages", ",".join(str(e) for e in languages)]
+        categories = ["Categories", ",".join(str(e) for e in categories)]
+
+        tags = [
+            characters,
+            parodies,
+            tags,
+            artists,
+            groups,
+            languages,
+            categories
+        ]
+        return tags
+
+    def embed(self, ctx, dic, DOUJIN_ID):
+        tags = self.doujin_tags(dic)
+
+        url = f"https://nhentai.net/g/{DOUJIN_ID}/"
+
+        if dic["images"]["pages"][0]["t"] == "j":
+            cover_url = f"https://i.nhentai.net/galleries/{dic['media_id']}/1.jpg"
+        elif dic["images"]["pages"][0]["t"] == "p":
+            cover_url = f"https://i.nhentai.net/galleries/{dic['media_id']}/1.png"
+
+        title = dic["title"]["pretty"]
+        num_pages = dic["num_pages"]
+
+        embed: discord.Embed = utils.Discord.create_embed(
+            author=ctx.message.author,
+            title=title,
+            url=url,
+            color=discord.Colour.from_rgb(227, 47, 86),
+        )
+        embed.set_image(url=cover_url)
+        embed.add_field(name="Number of pages", value=num_pages, inline=True)
+
+        for typ, tag in tags:
+            if tag != "":
+                if typ == "Tags":
+                    embed.add_field(name=typ, value=tag, inline=False)
+                else:
+                    embed.add_field(name=typ, value=tag, inline=True)
+        return embed
+
+    def is_channel_weeby(self, message):
+        if isinstance(message.channel, PrivateChannel):
+            return True
+        if message.channel.is_nsfw:
+            return True
+        return False
+
+    @commands.cooldown(rate=5, per=20.0, type=commands.BucketType.user)
+    @commands.command()
+    async def sauce(self, ctx):
+        """Give a magic number and you will get your sauce """
+        message = ctx.message
+        args = message.content.split(" ")
```

Comment:
Why are you getting args with split, instead of using the command argument parser?
```py
async def sauce(self, ctx, *args):
```